src/client-main.ts

```diff
@@ -786,7 +786,6 @@ const PS = new class extends PSModel {
 					}
 				}
 				this.update();
-				continue;
```

Comment:
?

Reply:
Don't remember why I removed this. I thought I had a good reason, but I don't think I did.